en/flight_modes/offboard.md

```diff
@@ -56,14 +56,28 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
 ### Fixed Wing
 
 * [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED)
-  * The following input combinations are supported: <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
+  * The following input combinations are supported (via `type_mask`): <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
     * Position setpoint (only `x`, `y`, `z`)
-  - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
-
+  * Specify the *type* of the setpoint in `type_mask` (not part of the MAVLink standard).
+    The values are:
+    - Following bits not set: vehicle will fly in a flower-like pattern.
+    - 4096: Takeoff setpoint.
+    - 8192: Land setpoint.

```

Comment:
Take off / Land type setpoints are largely untested. I will have a look if this is still valid.

Reply:
Thanks, waiting on you for that then! I really need to understand why all of these are actually useful to know - ie what does/can code do differently if it knows a setpoint is being used to land a vehicle? 

The reason I need this is that it will inform if the information is generically useful or just useful for PX4.

Reply:
@Jaeyoung-Lim Did you find out answer to this?